### PR TITLE
feat: allow customize meta in `nginx_config`

### PR DESCRIPTION
**Pull request by fengxsong**
if we configure prometheus plugin with extra labels like bellow

```
pluginAttrs:
  prometheus:
    metrics:
      http_status:
        extra_labels:
          - request_uri: $uri
          - request_method: $request_method
      http_latency:
        extra_labels:
          - request_uri: $uri
          - request_method: $request_method
```

then we might run into situation like https://github.com/apache/apisix/issues/7349.

this PR allow user to set the max size of `lua_shared_dict.prometheus-metrics`, incase we'll have more properties in this `meta` field in the future, so I make it as a string type.

### CODE REVIEW

**Comment by moonming** [comment]
ping @sn0rt please take a look,thx

**Comment by Sn0rt** [comment]
copy that. let me check pls assign this issues to me .

**Comment by fengxsong** [comment]
any updates?

File: charts/apisix/values.yaml
Comment (tao12345666333) [inline charts/apisix/values.yaml:393]: FYI： https://github.com/apache/apisix/blob/f2337f25d4bdbe7291810f36c0fbedd5cea8d29d/conf/config-default.yaml#L159-L161C24

**Review by tao12345666333 — APPROVED** [review]
LGTM

**Comment by Sn0rt** [comment]
LGTM

**Review by Sn0rt — APPROVED** [review]

**Review by moonming — APPROVED** [review]

**Comment by moonming** [comment]
@tao12345666333 can we merge this PR?

**Comment by tao12345666333** [comment]
sure. once we get enough approvers we can merge. no other blocking

**Pull request merged**
